src/main/target/AOCODARCF435V3/target.mk

```diff
@@ -0,0 +1,29 @@
+AT32F43xxGT7_TARGETS += $(TARGET) #FOR 1024KB
+
+CUSTOM_DEFAULTS_EXTENDED = yes
+FEATURES       +=  VCP ONBOARDFLASH
+
+TARGET_SRC = \
+			drivers/accgyro/accgyro_mpu6500.c \
+            drivers/accgyro/accgyro_spi_mpu6000.c \
+            drivers/accgyro/accgyro_spi_mpu6500.c\
+            drivers/accgyro/accgyro_spi_icm426xx.c\
```

Comment:
It seems that ICM42688P is supported, but last time you have mentioned that 42688 is not found. 

Any progress?

https://github.com/Aocoda-RC/atbetaflight/blob/efc7b2800b43994f52f4c148c0a03aebcbf64244/src/main/drivers/accgyro/accgyro_spi_icm426xx.c#L217-L234